src/test/java/com/logigear/RegisterTest.java

```diff
@@ -17,13 +17,34 @@
     @Test(description = "TC07 - User can create new account")
     public void TC07() {
         LoggerHelper.startTestCase("TC07 - User can create new account");
-        homePage.goToRegisterPage();
 
+        homePage.goToRegisterPage();
         Account account = new Account();
         registerPage.register(account);
 
         String actualMsg = registerPage.getSuccessfulMessage();
         String expectedMsg = "You're here";
         Assert.assertEquals(actualMsg, expectedMsg, "Error message fails to display!");
     }
+
+    @Test(description = "TC11 - User can't create account while password and PID fields are empty")
+    public void TC11() {
+        LoggerHelper.startTestCase("TC11 - User can't create account while password and PID fields are empty");
+
+        String expectedErrorMsg = "There're errors in the form. Please correct the errors and try again.";
+        String expectedErrorPasswordMsg = "Invalid password length";
+        String expectedErrorPidMsg = "Invalid ID length";
+
+        homePage.goToRegisterPage();
+        Account account = new Account(Constant.USERNAME, "", "");
+        registerPage.register(account);
+
+        String actualErrorMsg = registerPage.getGeneralErrorMessage();
+        String actualErrorPasswordMsg = registerPage.getPasswordErrorMessage();
+        String actualErrorPidMsg = registerPage.getPidErrorMessage();
+
+        Assert.assertEquals(actualErrorMsg, expectedErrorMsg, "Error message fails to display! ");
+        Assert.assertEquals(actualErrorPasswordMsg, expectedErrorPasswordMsg, "Error password message fails to display!");
```

Comment:
Error password message -> this is Vietnamese to English

Reply:
Thank you for the review, I will improve my grammar English every day